cartography/intel/gcp/crm/folders.py

```diff
@@ -37,68 +58,55 @@ def load_gcp_folders(
     neo4j_session: neo4j.Session,
     data: List[Dict],
     gcp_update_tag: int,
+    org_resource_name: str,
 ) -> None:
     """
-    Ingest the GCP folders to Neo4j.
+    Load GCP folders into the graph.
+    :param org_resource_name: Full organization resource name (e.g., "organizations/123456789012")
     """
+    # Transform data in place: add parent_org or parent_folder fields based on parent type
```

Comment:
Nit: not going to block on this, but it would be more consistent with other modules to create a separate transform function that operates on the data as a copy to keep a sort of functional style. I do like having transform funcs where it's easy later on to see what are the key names and values of each dict in the list. Again, not a blocker though.